documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -14,21 +14,6 @@ You must specify a name for cluster-scoped CRs.
 You must specify both a name and a namespace for namespace-scoped CRs.
 ====
 
-As a Technology Preview, {project-short} supports migrations using OpenStack source providers.
-
-[IMPORTANT]
-====
-Migration using OpenStack source providers is a Technology Preview feature only. Technology Preview features
-are not supported with Red Hat production service level agreements (SLAs) and
-might not be functionally complete. Red Hat does not recommend using them
-in production. These features provide early access to upcoming product
-features, enabling customers to test functionality and provide feedback during
-the development process.
-
-For more information about the support scope of Red Hat Technology Preview
-features, see https://access.redhat.com/support/offerings/techpreview/.
-====
-
 [NOTE]
 ====
 Migration using {osp} source providers only supports VMs that use only Cinder volumes.
```

Comment:
this reminds me that we added a ticket in Jira for QE but not for DOC that affects this part, done now: https://issues.redhat.com/browse/MTV-657

Reply:
@ahadas "Cinder volumes" is handled by https://github.com/kubev2v/forklift-documentation/pull/412.